### PR TITLE
[FLINK-6784][docs] update externalized checkpoints documentation

### PR DESCRIPTION
**Pull request by NicoK**
please also merge into `master`

### CODE REVIEW

**Comment by NicoK** [comment]
playing around with this some more, I'd like to add some more text about the directory structure and restore

File: docs/setup/checkpoints.md [hunk 1]
```diff
@@ -28,12 +28,21 @@ under the License.
 
 ## Overview
 
-TBD
-
+Checkpoints make state in Flink fault tolerant by allowing to recover state
+and positions in the streams to give the application the same semantics as
+a failure-free execution.
```
Comment (alpinegizmo) [inline docs/setup/checkpoints.md]: Checkpoints make state in Flink fault tolerant by allowing state
and the corresponding stream positions to be recovered, 
thereby giving the application the same semantics as a failure-free execution.

File: docs/setup/checkpoints.md [hunk 2]
```diff
@@ -54,4 +63,17 @@ state.checkpoints.dir: hdfs:///checkpoints/
 
 This directory will then contain the checkpoint meta data required to restore the checkpoint. The actual checkpoint files will still be available in their configured directory. You currently can only set this via the configuration files.
 
-Follow the [savepoint guide]({{ site.baseurl }}/setup/cli.html#savepoints) when you want to resume from a specific checkpoint.
+### Difference to Savepoints
+
+Externalized checkpoints have a few differences to [savepoints](savepoints.html). They
```
Comment (alpinegizmo) [inline docs/setup/checkpoints.md]: differences from

**Comment by NicoK** [comment]
Thanks, I addressed your comments and added some new sections which make the documentation more complete

**Comment by zentol** [comment]
+1 to merge.

**Comment by zentol** [comment]
merging.

**Comment by twalthr** [comment]
@zentol what is the status of this PR? It is still not merged.

**Pull request closed without merging**